src/psyclone/domain/common/algorithm/psyir.py

```diff
@@ -271,6 +278,31 @@ def lower_to_language_level(self):
         call = Call.create(routine_symbol, arguments)
         self.replace_with(call)
 
+        # Remove original invoke symbol if there are no other
```

Comment:
Quotes around 'invoke' please for clarity.

Reply:
Clariterised.